may_six/희수/퇴사.py

```diff
@@ -0,0 +1,16 @@
+import sys
+input = sys.stdin.readline
+n = int(input())
+
+dp = [0] * 20
+tplist = [[0, 0]]
+for _ in range(n):
+    t, p = map(int, input().split())
+    tplist.append([t, p])
+
+for i in range(1, n + 1):
+    x = tplist[i][0] - 1
+    dp[i] = max(dp[i - 1], dp[i])
+    dp[i + x] = max(dp[i - 1] + tplist[i][1], dp[i + x])
```

Comment:
```suggestion
    if i + x < 20:
        dp[i + x] = max(dp[i - 1] + tplist[i][1], dp[i + x])
```
(i + x)의 값이 20보다 작은지 확인하지 않으면 인덱스에러가 나는 경우가 발생할 수 있어요

문제에는 테스트 케이스가 부족해서 정답처리되었지만 아래와 같은 테스트 케이스가 있으면 오답이 될 수도 있어용

```
15
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
20 10
```